src/components/TokenClaimDialog.tsx

```diff
@@ -187,19 +202,41 @@ export default function TokenClaimDialog({
               </Trans>
             </Typography>
           )}
-          {allPoolsWithRewards.map((pool, i, arr) => (
-            <React.Fragment key={pool.poolName}>
-              <ClaimListItem
-                title={pool.poolName}
-                amount={rewardBalances[pool.poolName] || Zero}
-                claimCallback={() => claimPoolReward(pool)}
-                status={
-                  claimsStatuses["allPools"] || claimsStatuses[pool.poolName]
-                }
-              />
-              {i < arr.length - 1 && <Divider key={i} />}
-            </React.Fragment>
-          ))}
+          {gauges.map(
+            (gauge, i, arr) =>
+              userState?.gaugeRewards?.[gauge?.address]?.claimableSDL.gt(
+                Zero,
+              ) && (
+                <React.Fragment key={gauge?.poolName}>
+                  <ClaimListItem
+                    title={gauge.poolName}
+                    amount={
+                      userState?.gaugeRewards?.[gauge?.address]?.claimableSDL ??
+                      Zero
+                    }
+                    claimCallback={() => claimGaugeReward(gauge)}
+                    status={
+                      claimsStatuses["allGauges"] ||
+                      claimsStatuses[gauge.poolName]
+                    }
+                  />
+                  {i < arr.length - 1 && <Divider key={i} />}
+                </React.Fragment>
+              ),
+          )}
+          {/* {allPoolsWithRewards.map((pool, i, arr) => ( */}
+          {/*   <React.Fragment key={pool.poolName}> */}
+          {/*     <ClaimListItem */}
+          {/*       title={pool.poolName} */}
+          {/*       amount={rewardBalances[pool.poolName] || Zero} */}
+          {/*       claimCallback={() => claimPoolReward(pool)} */}
```

Comment:
@hammeiam safe to delete old pool list pointing to minichef?

Reply:
yep I think that should be fine! Exciting